src/btree/bt_curnext.c

```diff
@@ -679,6 +679,11 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
         page = cbt->ref == NULL ? NULL : cbt->ref->page;
 
         if (F_ISSET(cbt, WT_CBT_ITERATE_APPEND)) {
+            /*
+             * At this point there's no way the page to be NULL, but Coverity is unconvinced. So,
```

Comment:
I think we don't need to mention Coverity in the code. Just say the page cannot be null if we have the flag set.

Reply:
@quchenhao Fixed